foundry.toml

```diff
@@ -1,9 +1,35 @@
 [profile.default]
-src = 'src'
-out = 'out'
-libs = ['lib']
+  libs = ['lib']
```

Comment:
the `src`, `out`, and `libs` rows can be deleted since these are just the defaults